sdk/testing/src/main/java/io/opentelemetry/sdk/testing/assertj/LogRecordDataAssert.java

```diff
@@ -167,6 +172,97 @@ public LogRecordDataAssert hasBody(@Nullable Value<?> body) {
     return this;
   }
 
+  public LogRecordDataAssert hasBodyField(String key, String value) {
```

Comment:
Let's add javadoc for all these new public methods:
```suggestion
  /** Assert the log has a body of type {@link ValueType#KEY_VALUE_LIST}, containing a field with the given {@code key} and {@code value}. */
  public LogRecordDataAssert hasBodyField(String key, String value) {
```

Reply:
Yeah good call. I think they're pretty self-explanatory, but we have a policy of documenting public methods, even in test helper code.  Added. 👍🏻